packages/snaps-sdk/src/jsx/components/form/Selector.ts

```diff
@@ -0,0 +1,83 @@
+import type { SnapsChildren } from '../../component';
+import { createSnapComponent } from '../../component';
+import type { CardElement } from '../Card';
+
+/**
+ * The props of the {@link Selector.Option} component.
+ *
+ * @property value - The value of the selector option. This is used to populate the
+ * state in the form data.
+ * @property children - The component to display.
+ */
+export type SelectorOptionProps = {
+  value: string;
+  children: CardElement;
```

Comment:
Potentially this should be all components, undecided on whether to limit it for now